cgmanifest.json

```diff
@@ -998,7 +998,7 @@
                 "Type": "other",
                 "Other": {
                     "Name": "Docker Image: mcr.microsoft.com/vscode/devcontainers/java",
-                    "Version": "11-bullseye",
+                    "Version": "17-bullseye",
```

Comment:
@Chuxel Do you know what the version in cgmanifest.json for? Taken as default image version?

Reply:
@joshspicer @chrmarti Do you have insights here?

Reply:
This is used for registering OSS - it's auto-generate based on the contents of definition-manifest.json files. Changes will just be overwritten.

Reply:
I have removed this changed, as this is useless

Reply:
LGTM